cfgov/unprocessed/css/pages/beta.less

```diff
@@ -23,64 +23,64 @@
     .respond-to-min(@bp-sm-min, {
         font-size: 1em;
     });
-}
 
-.beta-banner_head {
-    .h4();
-    margin-bottom: 0;
+    &_head {
+        .h4();
+        margin-bottom: 0;
 
-    .m-expandable__expanded & {
-        margin-bottom: unit(5px / @base-font-size-px, em);
+        .m-expandable__expanded & {
+            margin-bottom: unit(5px / @base-font-size-px, em);
+        }
     }
-}
 
-.beta-banner_icon {
-    color: @gold;
-}
+    &_icon {
+        color: @gold;
+    }
 
-.beta-banner_desc {
-    margin-bottom: 0;
-    padding: 0;
-    background: @gold-20;
-}
+    &_desc {
+        margin-bottom: 0;
+        padding: 0;
+        background: @gold-20;
+    }
 
-.beta-banner_btn {
-    .btn__secondary();
-    width: 100%;
-    // Restore standard btn padding, which is removed by .expandable_target.
-    padding:
-        unit( @btn-v-padding / @btn-font-size, em )
-        unit( 14px / @btn-font-size, em );
-    margin-top: @grid_gutter-width / 2;
+    &_btn {
+        .btn__secondary();
+        width: 100%;
+        // Restore standard btn padding, which is removed by .expandable_target.
+        padding:
+            unit( @btn-v-padding / @btn-font-size, em )
+            unit( 14px / @btn-font-size, em );
+        margin-top: @grid_gutter-width / 2;
 
-    .respond-to-min(@bp-sm-min, {
-        .btn__link();
-        width: auto;
-        padding: @grid_gutter-width / 2;
-        margin: 0;
-        position: absolute;
-        top: 1px;
-        right: 0;
-        .u-link__colors(@black, @dark-gray);
-        .u-link__no-border();
-        text-transform: lowercase;
-    });
+        .respond-to-min(@bp-sm-min, {
+            .btn__link();
+            width: auto;
+            padding: @grid_gutter-width / 2;
```

Comment:
I just deleted the old home page stuff, this will all go away when the banner does, so I'm going to leave it for now.